app/src/main/java/com/kickstarter/viewmodels/CreatorDashboardViewModel.java

```diff
@@ -65,31 +59,25 @@ public ViewModel(final @NonNull Environment environment) {
       final Observable<Project> latestProject = projects
         .map(ListUtils::first);
 
+      final Observable<ProjectStats> projectStats = latestProject
+        .switchMap(this.client::fetchProjectStats);
```

Comment:
sweeet. for our api requests, we do have to handle the error state and call `.share()` to prevent too many requests from happening. we can handle the error by calling `.materialize()` after the request is made, which then gives us an `Observable<Notification<ProjectStats>>` that then holds either `values()` or `error()`

heres an example of [materialize and share in action](https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/viewmodels/CommentsViewModel.java#L95-L97) and here's how we can [extract the values](https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/viewmodels/CommentsViewModel.java#L99-L100) and the [errors](https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/viewmodels/CommentsViewModel.java#L139-L142)